templates/web/peterborough/email_sent.html

```diff
@@ -27,9 +27,17 @@ <h1>[% loc("Nearly done! Now check your email&hellip;") %]</h1>
 
     <p>
     [% IF email_type == 'problem' %]
-        [% loc("Click the link in our confirmation email to publish your problem.") %]
+        [% IF non_public %]
```

Comment:
Do we want to factor this bit out to a separate base template, given it is the same here and in base (and cheshireeast is going so doesn't matter, so only tfl would then need a different one)?